src/plugins/GUITestBase/src/GTUtilsDashboard.h

```diff
@@ -39,6 +39,32 @@ class GTUtilsDashboard {
                 Input,
                 ExternalTools };
 
+    // Dashboard notification class. Contains the type of notification (info/warning/error), the name of the workflow
+    // element to which the notification relates and the notification message
+    struct Notification {
+        // Type of notification
+        enum class Type { INFO,
```

Comment:
Declaring Notification.Type as enum in the test package + writing a framework how to convert it to string & back looks like an over-design.

More than that the "type" used in the U2Designer code as a pure QString.

I would suggest to remove this enum and use plain old QString for the "type" in tests.
For the known set of types we can have a static named constants: static QString Notification::warning = "warning".